packages/vscode/src/extension.ts

```diff
@@ -84,6 +84,27 @@ export async function activate(context: ExtensionContext): Promise<void> {
     },
   }
 
+  // enable fileWatcher to watch .prisma folder inside node_modules
+  const value: string = JSON.stringify(
+    workspace.getConfiguration().get('files.watcherExclude'),
```

Comment:
Is config `.get` method safe? Can it return null?

Reply:
I think it can be undefined (but there is a default variant where you can pass the default value as the second parameter)

https://github.com/microsoft/vscode/blob/master/src/vs/vscode.d.ts#L4754